src/formatter/javascript.ts

```diff
@@ -101,23 +101,15 @@ function toAssertCall(pageAlias: string, action: Action) {
 }
 
 function toSignalMap(action: Action) {
-  let waitForNavigation: Signal | undefined;
-  let assertNavigation: Signal | undefined;
   let popup: Signal | undefined;
   let download: Signal | undefined;
   let dialog: Signal | undefined;
   for (const signal of action.signals) {
-    if (signal.name === 'navigation' && signal.isAsync)
```

Comment:
[question]

Why are not waitForNavigation and assertNavigation needed anymore?

Reply:
Playwright action generator responds with waitForURL in these cases, We dont use it on the ActionGenerator.